pkg/conversion.go

```diff
@@ -72,10 +72,26 @@ func Conversion(KedgeSpecLocation, KubernetesSchema, OpenShiftSchema string) err
 	for k, v := range defs {
 		api.Schema.SchemaProps.Definitions[k] = v
 	}
+
+	if controllerOnly {
+		retainOnlyControllers(api.Schema.Definitions)
+	}
+
 	PrintJSONStdOut(api.Schema)
 	return nil
 }
 
+func retainOnlyControllers(definitions spec.Definitions) {
+	for k := range definitions {
+		switch k {
+		case "io.kedge.DeploymentSpecMod", "io.kedge.DeploymentConfigSpecMod", "io.kedge.JobSpecMod":
```

Comment:
going forward this should be configurable, like give it a config file and then here it is picked from that config!